includes/class-wc-calypso-bridge-customize-store.php

```diff
@@ -108,6 +113,47 @@ public function possibly_add_track_homepage_view() {
 		}
 	}
 
+	/**
+	 * Possibly enable Customize Store feature when feature flag is not already enabled
+	 * and experiment is treatment.
+	 * Criteria:
+	 *  1. Admin user
+	 *  2. Install date >= 2023-12-21 00:00:00 UTC
+	 */
+	public static function possibly_enable_cys( $features ) {
+		// When the feature is already enabled, return early since it's likely to be internal testing.
+		if ( isset( $features['customize-store'] ) ) {
+			return $features;
+		}
+
+		$timestamp = get_option( 'woocommerce_admin_install_timestamp', false );
+
+		// Eligiblity checks.
+		if ( self::is_admin() && $timestamp && $timestamp >= 1703116800 ) {
+			if ( class_exists( '\WooCommerce\Admin\Experimental_Abtest' ) && \WooCommerce\Admin\Experimental_Abtest::in_treatment( 'woocommerce_wooexpress_cys_launch_v1', true ) ) {
+				if ( ! in_array( 'customize-store', $features, true ) ) {
```

Comment:
Any reason to use `in_array` here and `isset` above? I don't remember the shape of `$features` off the top of my head, but it would seem if you use `in_array` above this check won't be necessary?

Reply:
AFAIK `$features` contains simple string in array, `['customize-store', ...]`.

You're right, `in_array` is unnecessary here.